src/libraries/System.Security.Principal.Windows/src/System/Security/Principal/SID.cs

```diff
@@ -394,7 +396,7 @@ private void CreateFromParts(IdentifierAuthority identifierAuthority, ReadOnlySp
             }
         }
 
-        private void CreateFromBinaryForm(byte[] binaryForm, int offset)
+        private void CreateFromBinaryForm(byte[]? binaryForm, int offset)
```

Comment:
The `binaryForm` parameter is immediately checked for null and throws if that is the case so this method will never accept a null value. Doesn't that mean that the type should be non-null? I realise that a nullable array is needed later but another variable could be used for that.

Reply:
You're right that the signature of the method should be updated to reflect that check.

Reply:
Did you change this? I can't see it but it might be me know knowing how to use GitHub.